### PR TITLE
[bitnami/fluentd] Chart standardized

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/fluentd chart to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/fluentd/values.yaml [hunk 1]
```diff
@@ -324,6 +340,38 @@ forwarder:
...
+    ## @param forwarder.service.sessionAffinity Session Affinity for Kubernetes service, can be "None" or "ClientIP"
+    ## If "ClientIP", consecutive client requests will be directed to the same Pod
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies
+    ##
+    sessionAffinity: None
+    ## @param forwarder.service.sessionAffinityConfig Additional settings for the sessionAffinity
+    ## sessionAffinityConfig:
+    ##   clientIP:
+    ##     timeoutSeconds: 300
+    sessionAffinityConfig: {}
```
Comment (carrodher) [inline bitnami/fluentd/values.yaml:352]:
```suggestion
    ##     timeoutSeconds: 300
    ##
    sessionAffinityConfig: {}
```

File: bitnami/fluentd/values.yaml [hunk 2]
```diff
@@ -494,7 +571,7 @@ forwarder:
   ## preStop:
   ##   exec:
   ##     command: ["/bin/sh","-c","nginx -s quit; while killall -0 nginx; do sleep 1; done"]
-  lifecycle: {}
+  lifecycleHooks: {}
```
Comment (carrodher) [inline bitnami/fluentd/values.yaml:574]:
```suggestion
  ##
  lifecycleHooks: {}
```

File: bitnami/fluentd/values.yaml [hunk 3]
```diff
@@ -983,7 +1156,7 @@ aggregator:
...
```
Comment (carrodher) [inline bitnami/fluentd/values.yaml:1159]:
```suggestion
  ##
  lifecycleHooks: {}
```

File: bitnami/fluentd/values.yaml [hunk 4]
```diff
@@ -1048,12 +1212,38 @@ metrics:
...
+    ## @param metrics.service.sessionAffinity Session Affinity for Kubernetes service, can be "None" or "ClientIP"
+    ## If "ClientIP", consecutive client requests will be directed to the same Pod
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies
+    ##
+    sessionAffinity: None
+    ## @param metrics.service.sessionAffinityConfig Additional settings for the sessionAffinity
+    ## sessionAffinityConfig:
+    ##   clientIP:
+    ##     timeoutSeconds: 300
+    sessionAffinityConfig: {}
```
Comment (carrodher) [inline bitnami/fluentd/values.yaml:1246]:
```suggestion
    ##     timeoutSeconds: 300
    ##
    sessionAffinityConfig: {}
```

File: bitnami/fluentd/values.yaml [hunk 5]
```diff
@@ -958,23 +1113,41 @@ aggregator:
...
+    ## @param aggregator.persistence.storageClass Persistent Volume storage class
     ## If defined, storageClassName: <storageClass>
     ## If set to "-", storageClassName: "", which disables dynamic provisioning
     ## If undefined (the default) or set to null, no storageClassName spec is
     ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
     ##   GKE, AWS & OpenStack)
     ##
     storageClass: ""
-    accessMode: ReadWriteOnce
+    ## @param aggregator.persistence.accessModes [array] Persistent Volume access modes
```
Comment (carrodher) [inline bitnami/fluentd/values.yaml:1129]:
```suggestion
    ## @param aggregator.persistence.accessModes Persistent Volume access modes
```

File: bitnami/fluentd/values.yaml [hunk 6]
```diff
@@ -1019,16 +1192,7 @@ aggregator:
...
-## @param serviceAccount Pods Service Account. This top-level global entry is DEPRECATED. Please use "forwarder.serviceAccount" instead.
-## Only the forwarder was affected by the historical usage here.
-## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
-##
-serviceAccount: {}
-## @param rbac Role Based Access. This top-level global entry is DEPRECATED. Please use "forwarder.rbac" instead.
-## Only the forwarder was affected by the historical usage here.
-## ref: https://kubernetes.io/docs/admin/authorization/rbac/
-##
-rbac: {}
```
Comment (juan131) [inline bitnami/fluentd/values.yaml:1031]: Why do these parameters disappear?

**Review by juan131 — CHANGES_REQUESTED** [review]

File: bitnami/fluentd/values.yaml (hunk 6 above)
Comment (migruiz4) [reply bitnami/fluentd/values.yaml:1031]: They were already deprecated and removed from the code. They have been for a long time under `forwarder.*`.

If set, they would trigger a warning (Check _helpers.tpl).

I thought about removing those messages too, but at least it doesn't make sense having them in the `values.yaml` in my opinion.

Comment (juan131) [reply bitnami/fluentd/values.yaml:1031]: Please go ahead and remove the warnings from the helpers too

Comment (migruiz4) [reply bitnami/fluentd/values.yaml:1031]: Done 👍

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**